analytics/sensors/src/main/java/quarks/analytics/sensors/Filters.java

```diff
@@ -114,4 +114,27 @@ private Filters() {}
 
         return stream.filter(new Deadband<>(value, inBand));
     }
+    
+    /**
+     * A filter that discards tuples received during the "deadtime period"
```

Comment:
Not English! and too long for the summary that is seen in Javadoc.

Maybe just:

Deadtime filter.

(c.f. Deadband filter)

Then have the following sentences describe what it does.